apps/docs/package.json

```diff
@@ -7,7 +7,8 @@
     "build": "npm run clean && cross-env NODE_ENV=production gatsby build --prefix-paths",
     "build:preview": "npm run clean && cross-env NODE_ENV=production gatsby build",
     "clean": "rm -rf public && rm -rf .cache",
-    "start": "rm -rf .cache && gatsby develop --open"
+    "start": "rm -rf .cache && gatsby develop --open",
+    "build:animation": "node scripts/animation-support.js"
```

Comment:
Maybe worth executing this `npm script` on every `build`? Otherwise, there is a possibility the list will get outdated if we forget to run the script manually.

Reply:
good point, done 👍